server/application/application_test.go

```diff
@@ -3350,3 +3351,106 @@ func Test_RevisionMetadata(t *testing.T) {
 		})
 	}
 }
+
+func Test_DeepCopyInformers(t *testing.T) {
```

Comment:
This is the "real" test. Ensuring the deep copy informers return an actual deep copy.